AnsibleGalaxyContentQualityScore service test failing

:clock11: **When did the problem start?**
~3 days ago, 1st of July 

<!-- Indicate when the problem started -->

:camera: **Live badge**

<!-- Provide a link to the live badge in plain text and markdown.  -->
https://img.shields.io/ansible/quality/432
![](https://img.shields.io/ansible/quality/432)

:wrench: **Is the live badge working?**
Yes
<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://app.circleci.com/pipelines/github/badges/daily-tests/318/workflows/079cd9dc-054d-4ecb-ac88-7cd50cd24da5/jobs/1215
<!-- Provide a link to the failing test in CircleCI. -->

:beetle: **Stack trace**

```
AnsibleGalaxyContentQualityScore [live] quality score (not found) [ GET /0101.json ] - [ GET /0101.json ]
/home/circleci/project/shields/core/service-test-runner/cli.js

AssertionError: message mismatch: expected 'not found' to equal 'no score available'
    at Function._expectField (core/service-test-runner/icedfrisby-shields.js:69:51)
    at IcedFrisbyNock.<anonymous> (core/service-test-runner/icedfrisby-shields.js:54:26)
    at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:954:10)
    at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:1003:24)
    at /home/circleci/project/shields/node_modules/icedfrisby/lib/icedfrisby.js:1017:7
    at new Promise (<anonymous>)
    at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:976:12)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1276:20)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Context.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:1348:9)
```

:bulb: **Possible solution**
Not sure yet
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/ansible/ansible-quality.tester.js
+++ b/services/ansible/ansible-quality.tester.js
@@ -7,6 +7,10 @@ t.create('quality score (valid)')
   .get('/432.json')
   .expectBadge({ label: 'quality', message: nonNegativeInteger })
 
-t.create('quality score (not found)')
+t.create('quality score (project not found)')
   .get('/0101.json')
+  .expectBadge({ label: 'quality', message: 'not found' })
+
+t.create('quality score (no score available)')
+  .get('/2504.json')
   .expectBadge({ label: 'quality', message: 'no score available' })

